File: StudyJet.UI/src/app/models/user/login-response.dto.ts

export interface LoginResponse {
    requires2FA?: boolean;
    tempToken?: string;
    requiresPasswordChange?: boolean;
    resetToken?: string;
    token?: string;
    roles?: string[];
    username?: string;
    profilePictureUrl?: string;
    fullName?: string;
    userId?: string;
  }
  

File: StudyJet.API/DTOs/User/LoginResponseDTO.cs
namespace StudyJet.API.DTOs.User
{
    public class LoginResponseDTO
    {
        public string Token { get; set; }
        public List<string> Roles { get; set; }

        public string ProfilePictureUrl { get; set; }

        public string UserName { get; set; }

        public string FullName { get; set; }

        public string Email { get; set; }

        public string UserID { get; set; }
    }
}
